packages/connector-kakao/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@logto/connector-kakao",
+  "version": "1.0.0-beta.5",
+  "description": "Kakao connector implementation.",
+  "main": "./lib/index.js",
+  "exports": "./lib/index.js",
+  "author": "Kyungyoon Kim. <ruddbs5302@gmail.com>",
+  "license": "MPL-2.0",
+  "private": true,
+  "files": [
+    "lib",
+    "docs",
+    "logo.svg",
+    "README.md"
+  ],
+  "scripts": {
+    "precommit": "lint-staged",
+    "build": "rm -rf lib/ && tsc -p tsconfig.build.json",
+    "dev": "tsc -p tsconfig.build.json --watch --preserveWatchOutput --incremental",
+    "lint": "eslint --ext .ts src",
+    "lint:report": "pnpm lint --format json --output-file report.json",
+    "test": "jest",
+    "test:coverage": "jest --coverage --silent",
+    "prepack": "pnpm build"
+  },
+  "dependencies": {
+    "@logto/connector-core": "^1.0.0-beta.5",
+    "@logto/shared": "^1.0.0-beta.5",
```

Comment:
Maybe we can remove this dependency as we have already removed this dep from other connectors.
```suggestion
```